platform/metrics/platform-metrics-reporting/pom.xml

```diff
@@ -101,7 +101,7 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>rrd4j,joda-time, commons-math, platform-util, poi-scratchpad
+                        <Embed-Dependency>rrd4j,joda-time, commons-math, platform-util, poi-scratchpad, poi
```

Comment:
A comment on the other ticket indicates this could just be a version range issue. It looks like poi is available in the container, should we just correct the version range on the import instead of embedding it? https://codice.atlassian.net/browse/DDF-3879?focusedCommentId=32602&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-32602

Reply:
That wasn't what I was seeing when inspecting the external libraries after a clean build. We already specify the correct version in our dependencies. Maybe I'm confused with what you are suggesting.